katello_certs_tools/katello_ssl_tool.py

```diff
@@ -739,48 +777,50 @@ def genProxyServerTarball(d, version='1.0', release='1', verbosity=0):
     tarballFilepath2 = pathJoin(serverKeySetDir, tarballFilepath)
 
     if verbosity >= 0:
-        print """
+        print("""
 The most current RHN Proxy Server installation process against RHN hosted
 requires the upload of an SSL tar archive that contains the CA SSL public
 certificate and the web server's key set.
 
 Generating the web server's SSL key set and CA SSL public certificate archive:
-    %s""" % tarballFilepath2
+    %s""" % tarballFilepath2)
 
     cwd = chdir(d['--dir'])
     try:
         if verbosity > 1:
-            print "Current working directory:", os.getcwd()
-            print "Commandline:", args
+            print("Current working directory:", os.getcwd())
+            print("Commandline:", args)
         ret, out_stream, err_stream = rhn_popen(args)
     finally:
         chdir(cwd)
 
-    out = out_stream.read(); out_stream.close()
-    err = err_stream.read(); err_stream.close()
+    out = out_stream.read()
+    out_stream.close()
+    err = err_stream.read()
+    err_stream.close()
 
     if ret or not os.path.exists(tarballFilepath):
         raise GenServerTarException(
-	  "CA SSL public certificate & web server's SSL key set tar archive\n"
-	  "generation failed:\n%s\n%s" % (out, err))
+            "CA SSL public certificate & web server's SSL key set tar archive\n"
+            "generation failed:\n%s\n%s" % (out, err))
     if verbosity > 2:
         if out:
-            print "STDOUT:", out
+            print("STDOUT:", out)
         if err:
-            print "STDERR:", err
+            print("STDERR:", err)
 
     # root baby!
-    os.chmod(tarballFilepath, 0600)
+    os.chmod(tarballFilepath, '0600')
```

Comment:
`os.chmod` does not accept string parameters (at least not in Python 3.6 and 3.7)